src/science/landuse/landuse3.F90

```diff
@@ -1102,6 +1129,16 @@ SUBROUTINE landuse_mp2land(luc,lucmp,mp,cstart,cend)
 END SUBROUTINE landuse_mp2land
   
 SUBROUTINE landuse_transitx(luc,casabiome)
+   !* This subroutine calculates: 
+   !
+   ! 1. the transfer of different C, N and P pools resulting from land use
+   ! change in two steps
+   !     1. calculate the change of a pool (`delvar`) using transition 
+   !     matrix (`T`)
+   !     1. calculate `luc%var_y` as the sum of `luc%var_x + delvar`
+   ! 2. update biophysical states, soil texture and soil order for each patch
+   ! 3. seed any deforested land
+   !
```

Comment:
```suggestion
   !*## Purpose
   !
   ! This subroutine applies the land use changes to the transfer of the C, N and P pools, the biophysical states, 
   ! the soil texture and the soil order for each patch. Then it will see the deforested land.
   !
   !## Order of procedure
   !
   ! 1. the transfer of different C, N and P pools resulting from land use
   ! change in two steps
   !     1. calculate the change of a pool (`delvar`) using transition 
   !     matrix (`T`)
   !     1. calculate `luc%var_y` as the sum of `luc%var_x + delvar`
   ! 2. update biophysical states, soil texture and soil order for each patch
   ! 3. seed any deforested land
   !
```
Again just a suggestion for structure.